src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs

```diff
@@ -747,26 +747,31 @@ internal void PrepareForBind(OdbcCommand command, short ordinal, ref int paramet
                 }
             };
 
-            int cbParameterSize = GetParameterSize(value, offset, ordinal);      // count of bytes for the data, for SQLBindParameter
+            int cbParameterSize = GetParameterSize(value, offset, ordinal); // count of bytes for the data, for SQLBindParameter
```

Comment:
Move this GetParameterSize() line to after the switch statement, since we no longer use cbParameterSize in the switch.